DESCRIPTION

```diff
@@ -83,7 +83,10 @@ Imports:
     scales,
     ggplot2,
     posterior,
-    MASS
+    MASS,
+    cramer,
+    expm,
+    rstan
```

Comment:
If `rstan` is just used in the `scratch` or even in the `vignettes`, as here, it can be in `Suggests`. But any particular reason why you used `rstan` versus `cmdstanr` for those `Rmd` examples?

Reply:
I did not know that cmdstanr was an alternate to rstan.  I can switch them.

Reply:
From my memory, the syntax for writing an example stan file directly in R is nicer in `rstan` ! Let me see if I can find the syntax of cmdstanr. You can't just swap them in and out, I don't think

Reply:
I can't seem to find it, but I am 90% positive its more annoying than just putting it in quotations...

Reply:
`cmdstanr` provides a [`write_stan_file`](https://mc-stan.org/cmdstanr/reference/write_stan_file.html) function for this use case.